src/main/java/com/redhat/cloud/notifications/routers/models/UserConfigPreferences.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.routers.models;
+
+public class UserConfigPreferences {
```

Comment:
Snake case should probably be enforced here with `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)`.

Reply:
This doesn't seem to update the openapi.json file accordingly.
My tests shows that the received data is on snake_case, but openapi shows camelCase.

Reply:
Not something urgent, as the API is hidden anyway.

Reply:
I ran into the same issue with the Hibernate Reactive branch. I couldn't find an application-wide setting so I ended up using `@Schema(name = "some_snake_case_name")` on each attribute with a camel case name.

Reply:
I ran into other issue with the `@RestClient` in policies code (it calls this service).  
The return value doesn't seem to honor the `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)` or the `@Schema(name = "some_snake_case_name")`.

Reply:
I just found an issue about the snake case strategy support during OpenAPI files generation: https://github.com/smallrye/smallrye-open-api/issues/712.